services/server/src/sourcify-chains.ts

```diff
@@ -238,14 +247,14 @@ for (const chain of allChains) {
       }
     });
```

Comment:
It's not strictly related to this PR, but I think it makes sense to log if the process.env variable doesn't exist so it's clear the the header value was set to "" because a missing var